### PR TITLE
Home game

### PR DESCRIPTION
**Pull request by Gonaline**

### CODE REVIEW

**Review by ValentinCop — DISMISSED** [review]

**Review by LuigiMenez — DISMISSED** [review]
Warning a blank Line

**Review by n-faycel — APPROVED** [review]

File: frontend/src/pages/Game/style.jsx [hunk 1]
```diff
@@ -1 +1,20 @@
...
+  .dice {
+    object-position: top;
+    z-index: 2;
+    transform: rotate(20deg);
+    position: relative;
+    bottom: 50px;
+  }
+  .pawn {
+    position: absolute;
+    z-index: 100;
```
Comment (LoicBrassart) [inline frontend/src/pages/Game/style.jsx:15]: ;_;

**Review by LoicBrassart — APPROVED** [review]
Beware of z-index, it's behaviour is a bit complex and can lead to errors

**Review by LuigiMenez — APPROVED** [review]
Ok

**Pull request merged**